misc/man/texmacs.1.in

```diff
@@ -1,14 +1,14 @@
 .TH texmacs 1 "12Sep2005" @PACKAGE@-@DEVEL_VERSION@
 .SH NAME
-GNU TeXmacs - a WYSIWYG mathematical text editor
+texmacs - GNU project TeXmacs WYSIWYG scientific text editor
```

Comment:
I think the title is OK

Reply:
Do you mean that the title is okay for humans ?